multiqc/modules/dragen/coverage_hist.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import itertools
+import math
```

Comment:
Is this used? I can't see it anywhere. Same for `itertools`.

```console
$ importchecker multiqc/modules/dragen/
multiqc/modules/dragen/coverage_hist.py:4: itertools
multiqc/modules/dragen/coverage_hist.py:5: math
multiqc/modules/dragen/coverage_hist.py:7: OrderedDict
multiqc/modules/dragen/coverage_hist.py:10: bargraph, table
multiqc/modules/dragen/coverage_metrics.py:6: math
multiqc/modules/dragen/coverage_metrics.py:8: OrderedDict
multiqc/modules/dragen/coverage_metrics.py:10: linegraph, bargraph
multiqc/modules/dragen/coverage_per_contig.py:4: itertools
multiqc/modules/dragen/coverage_per_contig.py:5: math
multiqc/modules/dragen/coverage_per_contig.py:9: bargraph
multiqc/modules/dragen/fragment_length.py:5: OrderedDict
multiqc/modules/dragen/mapping_metrics.py:7: OrderedDict
multiqc/modules/dragen/mapping_metrics.py:8: config
multiqc/modules/dragen/mapping_metrics.py:10: base_format, read_format
multiqc/modules/dragen/mapping_metrics.py:11: beeswarm
multiqc/modules/dragen/ploidy_estimation_metrics.py:7: linegraph
multiqc/modules/dragen/time_metrics.py:7: Metric
multiqc/modules/dragen/time_metrics.py:8: linegraph, table, beeswarm
multiqc/modules/dragen/vc_metrics.py:5: OrderedDict, defaultdict
multiqc/modules/dragen/vc_metrics.py:7: linegraph
```

([importchecker](https://pypi.org/project/importchecker/) is a nice little tool!)

Reply:
I'm actually using PyCharm, it highlights unused imports right in the editor :) 

Removed unused ones now.

Reply:
Hmm, I wonder if there's an Atom plugin to do this.. 🤔